app/signin/page.tsx

```diff
@@ -0,0 +1,11 @@
+import Auth from "@/src/components/Auth/Auth";
+import AuthHookForm from "@/src/components/Auth/AuthHookForm";
+
+export default function signinPage() {
+  return (
+    <>
+      {/* <Auth /> */}
+      <AuthHookForm />
```

Comment:
실제로 내부에서 어떤식으로 구현이 되어 있는지는 컴포넌트의 관심사가 아닌 것 같아요~
컴포넌트의 이름을 표현할때 해당 UI의 생김새, 목적을 드러내면 좋을 것 같습니다!

이렇게 표현해줘도 괜찮을 것 같네요~
```suggestion
      <SignInForm />
```